xml/System/UIntPtr.xml

```diff
@@ -1041,10 +1041,19 @@
         <Parameter Name="other" Type="System.UIntPtr" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="other">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="other">An unsigned integer pointer to compare with this instance.</param>
+        <summary>Returns a value that indicates whether this instance is equal to another unsigned integer pointer.</summary>
+        <returns>
+          <see langword="true" /> if <paramref name="other" /> equals the value of this instance; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This member is an explicit interface member implementation. It can be used only when the <xref:System.UIntPtr> instance is cast to an <xref:System.IEquatable> interface.
```

Comment:
```suggestion
This member is an explicit interface member implementation. It can be used only when the <xref:System.UIntPtr> instance is cast to an <xref:System.IEquatable%601> interface.
```